### PR TITLE
[0583] Course search filters sorted by

### PR DESCRIPTION
**Pull request by defong**
### Context
Sorted by

### Changes proposed in this pull request
Get sorted

### Guidance to review

https://publish-teacher-training-pr-3157.london.cloudapps.digital/find/results

image

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by defong** [comment]
[Course Search Filters - Sorted by](https://trello.com/c/etTa1TTm/583-course-search-filters-sorted-by)

File: app/services/course_search_service.rb [hunk 1]
```diff
@@ -10,8 +12,12 @@ def initialize(
   end
 
   class << self
-    def call(...)
-      new(...).call
+    def provider_ascending
```
Comment (darokel) [inline app/services/course_search_service.rb:15]: This seems a little redundant as class methods. Seems to only be used internally? They're also immutable - feels like they would fit better as constants in the class?

Comment (defong) [reply app/services/course_search_service.rb]: Good call done in https://github.com/DFE-Digital/publish-teacher-training/pull/3157/commits/d8724089fc9f5e26c3c7c589b83dc7b8a600452d

**Review by darokel — APPROVED** [review]

**Review by victoriawinlow — APPROVED** [review]
All good, thanks Sau

**Pull request merged**